pkgs/by-name/co/copywrite/package.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  testers,
+  copywrite,
+}:
+
+let
+  commitHash = "6ed520a710166c6094098b786c63f212604654a4"; # matches tag release
+  shortCommitHash = builtins.substring 0 7 commitHash;
+in
+buildGoModule rec {
+  pname = "copywrite";
+  version = "0.19.0";
+
+  src = fetchFromGitHub {
+    owner = "hashicorp";
+    repo = "copywrite";
+    rev = "v${version}";
+    hash = "sha256-DmlPioaw/wMk8GoBYNG24P4J1C6h0bjVjjOuMYW6Tgo=";
+  };
+
+  vendorHash = "sha256-ZIu0/fue3xi+YVE9GFsVjCNs8t3c3TWH8O0xUzJdim8=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/hashicorp/copywrite/cmd.version=${version}"
+    "-X github.com/hashicorp/copywrite/cmd.commit=${shortCommitHash}"
+  ];
+
+  CGO_ENABLED = 0;
```

```suggestion
  env.CGO_ENABLED = 0;
```
This was recently changed from a `buildGoModule` parameter to normal env variable.
ofborg would complain if it rebuilt the package.

---

Other than that the PR is ready to merge I think.

---

Done! Thanks a lot for the review.